src/wasm-lib/kcl/src/parsing/token/tokeniser.rs

```diff
@@ -54,9 +54,10 @@ lazy_static! {
         set.insert("string", TokenType::Type);
         set.insert("number", TokenType::Type);
         set.insert("bool", TokenType::Type);
-        set.insert("sketch", TokenType::Type);
-        set.insert("sketch_surface", TokenType::Type);
-        set.insert("solid", TokenType::Type);
+        set.insert("Sketch", TokenType::Type);
+        set.insert("SketchSurface", TokenType::Type);
+        set.insert("Solid", TokenType::Type);
+        set.insert("Plane", TokenType::Type);
```

Comment:
FYI, this is a breaking change since it's a parse error to define anything matching these reserved words.

We need to update the lezer grammar. Should be one line in `kcl.grammar`.